scripts/templates/webdriverio.tpl.d.ts

```diff
@@ -160,8 +160,9 @@ declare namespace WebdriverIO {
     interface Browser<T> {
         addCommand(
             name: string,
-            func: Function
-        ): any;
+            func: Function,
+            attachToElement?: boolean
```

Comment:
Incase it matters, I don't think this boolean exists at the element level

Reply:
good point, completely forgot about element. I mean this: `$('body').addCommand(...)`

Reply:
https://github.com/webdriverio/webdriverio/pull/3343/files#r248081983